torch/csrc/jit/codegen/cuda/expr_simplifier.cpp

```diff
@@ -194,9 +194,9 @@ std::unordered_set<Val*> getSubexprDependency(
   return result;
 }
 
-bool hasTensor(const std::unordered_set<Val*>& variables) {
+bool hasTensorIndex(const std::unordered_set<Val*>& variables) {
```

Comment:
Can you bury this as a lambda in its use? Uncertain why it's not just using `std::any_of`

Reply:
fixed